app/views/pdfjs_viewer/viewer/_viewer.html.erb

```diff
@@ -42,6 +42,7 @@ See https://github.com/adobe-type-tools/cmap-resources
       </script>
     <% end %>
     <%= javascript_include_tag "pdfjs_viewer/application" %>
+    <%= javascript_tag "window.HOSTED_VIEWER_ORIGINS = #{PdfjsViewer.hosted_viewer_origins.to_json.html_safe};" %>
```

Comment:
It's already configured in the browser isn't it?

> On 4 Aug 2016, at 08:23, Yves Senn <notifications@github.com> wrote:
> 
> In app/views/pdfjs_viewer/viewer/_viewer.html.erb:
> 
> > @@ -42,6 +42,7 @@ See https://github.com/adobe-type-tools/cmap-resources
> >        </script>
> >      <% end %>
> >      <%= javascript_include_tag "pdfjs_viewer/application" %>
> > +    <%= javascript_tag "window.HOSTED_VIEWER_ORIGINS = #{PdfjsViewer.hosted_viewer_origins.to_json.html_safe};" %>
> I don't think configuring the viewer in the browser is a good idea. This check is a safeguard to prevent malicious requests. The browser environment is not something to be trusted.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>